File: ArcherApp/app/src/main/java/me/mxcsyounes/archernotebook/DataProvider.java
package me.mxcsyounes.archernotebook;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

import me.mxcsyounes.archernotebook.database.entities.Adjustment;

public final class DataProvider {

    private static List<Adjustment> adjustmentList = new ArrayList<>();

    static {
        adjustmentList.add(new Adjustment(1, 2, "6.2", "1.3", new Date(),
                null, "Hello world"));
        adjustmentList.add(new Adjustment(1, 1, "11", "1.2", new Date(),
                null, "Hello algeria"));
        adjustmentList.add(new Adjustment(1, 3, "8", "4", new Date(),
                null, "Hello merrakch"));
        adjustmentList.add(new Adjustment(1, 4, "6", "12", new Date(),
                null, "Hello tiaret"));


    }

    public static List<Adjustment> getAdjustmentList() {
        return adjustmentList;
    }
}

File: ArcherApp/app/src/main/java/me/mxcsyounes/archernotebook/DataProvider.kt
package me.mxcsyounes.archernotebook

import me.mxcsyounes.archernotebook.database.entities.Adjustment
import java.util.*

object DataProvider {

    private val adjustmentList = ArrayList<Adjustment>()

    init {
        adjustmentList.add(Adjustment(1, 2, "6.2", "1.3", Date(),
                null!!, "Hello world"))
        adjustmentList.add(Adjustment(1, 1, "11", "1.2", Date(),
                null!!, "Hello algeria"))
        adjustmentList.add(Adjustment(1, 3, "8", "4", Date(),
                null!!, "Hello merrakch"))
        adjustmentList.add(Adjustment(1, 4, "6", "12", Date(), null!!,
                "Hello tiaret"))
    }
}
